AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginIntegrationTests/DataStoreLocalStoreTests.swift

```diff
@@ -421,6 +421,58 @@ class DataStoreLocalStoreTests: LocalStoreIntegrationTestBase {
         sink.cancel()
     }
 
+    func testDeleteModelTypeWithPredicate() {
+        _ = setUpLocalStore(numberOfPosts: 5)
+        let deleteSuccess = expectation(description: "Delete all successful")
```

Comment:
Can we have an `assert` before `delete` on the whether number of posts in draft status are > 0?

Reply:
good idea, updating!